pkg/ingester/ingester.go

```diff
@@ -413,8 +414,8 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	ingestionRateTicker := time.NewTicker(instanceIngestionRateTickInterval)
 	defer ingestionRateTicker.Stop()
 
-	exemplarUpdateTicker := time.NewTicker(i.cfg.ExemplarsUpdatePeriod)
-	defer exemplarUpdateTicker.Stop()
+	tsdbUpdateTicker := time.NewTicker(i.cfg.ExemplarsUpdatePeriod)
```

Comment:
Whould we state in the config docs that this period now serves for more than just updating exemplars?

Reply:
That's a good point. Shall we rename it too?

Reply:
It's still experimental after all.

Reply:
So I guess we agree on renaming it?

Reply:
I believe so.

Reply:
We need to mention that in CHANGELOG.